pkg/printers/internalversion/printers_test.go

```diff
@@ -548,8 +549,8 @@ func TestPrinters(t *testing.T) {
 	}
 
 	genericPrinters := map[string]printers.ResourcePrinter{
-		"json":      printers.NewTypeSetter(legacyscheme.Scheme).ToPrinter(&printers.JSONPrinter{}),
-		"yaml":      printers.NewTypeSetter(legacyscheme.Scheme).ToPrinter(&printers.YAMLPrinter{}),
+		"json":      genericprinters.NewTypeSetter(legacyscheme.Scheme).ToPrinter(&genericprinters.JSONPrinter{}),
```

Comment:
This is a sign that your unit tests are in the wrong spot.  after freeze, fix this debt.

Reply:
Thanks, added a todo here